docs/cards.html

```diff
@@ -0,0 +1,662 @@
+      <div class="container">
+        <h3 class="sr-only">Breadcrumbs</h3>
+        <div class="row">
+          <div class="col-sm-24">
+            <ol class="breadcrumb">
+              <li><a href="/">Home</a></li>
+              <li class="active">Eclipse Solstice</li>
+            </ol>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- /#breadcrumb -->
+    <main>
+      <div class="novaContent container legacy-page" id="novaContent">
+        <!-- Cards start-->  
+        <!--
+                <div class="glyph-highlight">
+                  <div class="glyph-container">
+                    <span class="padding-20">
+                      <i class="fa fa-globe"></i>
+                    </span>
+                  </div>
+                  <div class="glyph-bottom"></div>
+                </div>
+        -->
+        <div class="row">
+          <div class="col-xs-24">
+            <h2>Standard Image Cards</h2>
+            <div class="row">
+              <div class="card-container col-xs-24 col-sm-12 match-height-item-by-row">
+                <div class="card-panel panel panel-default">
+                  <div class="panel-heading" aria-hidden="true">
+                    <img class="img cover-image" src="dist/images/template/placeholders/500x500.png" />
+                  </div>
+                  <div class="panel-body">
+                    <h3 class="fw-700">Lorem Ipsum</h3>
+                    <p>
+                      Esse incididunt ipsum nulla consectetur tempor elit. Mollit adipisicing esse labore consectetur irure ut reprehenderit sit.
+                      Quis do cupidatat qui quis nostrud laboris nulla incididunt laborum sit sit qui id cupidatat.
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="card-container col-xs-24 col-sm-12 match-height-item-by-row">
+                <div class="card-panel panel panel-default">
+                  <div class="panel-heading" aria-hidden="true">
+                    <img class="img cover-image rounded" src="dist/images/template/placeholders/500x500.png" />
+                  </div>
+                  <div class="panel-body">
+                    <h3 class="fw-700">Lorem Ipsum</h3>
+                    <p>
+                      Esse incididunt ipsum nulla consectetur tempor elit. Mollit adipisicing esse labore consectetur irure ut reprehenderit sit.
+                      Quis do cupidatat qui quis nostrud laboris nulla incididunt laborum sit sit qui id cupidatat.
+                    </p>
+                  </div>
+                </div>
+              </div>
+              <div class="card-container col-xs-24 col-sm-12 match-height-item-by-row">
+                <div class="card-panel panel panel-default">
+                  <div class="panel-body">
+                    <img class="img img-responsive margin-auto" src="dist/images/template/placeholders/250x250.png" />
```

Comment:
Not sure if this is on purpose?
I think this is the only `img tag` that is put inside a `panel-body div`, all the others are in `panel-heading div`.